### PR TITLE
Simplify `F.ceil` test

### PR DESCRIPTION
**Pull request by hikjik**
Simplifies test_ceil using testing.FunctionTestCase
Refer to [#6071](https://github.com/chainer/chainer/issues/6071)

### CODE REVIEW

File: tests/chainer_tests/functions_tests/math_tests/test_ceil.py [hunk 1]
```diff
@@ -1,58 +1,53 @@
...
+class TestCeil(testing.FunctionTestCase):
+
+    skip_backward_test = True
+    skip_double_backward_test = True
+
+    def generate_inputs(self):
+        while True:
+            x = numpy.random.uniform(
+                -10.0, 10.0, self.shape).astype(self.dtype)
+            if (numpy.abs(x - numpy.round(x)) > 1e-2).all():
```
Comment (Crissman) [inline tests/chainer_tests/functions_tests/math_tests/test_ceil.py]: Can you comment on why you added this variance check?

Comment (hikjik) [reply tests/chainer_tests/functions_tests/math_tests/test_ceil.py]: This distance check is used for stability. Please take a look here [#3427](https://github.com/chainer/chainer/pull/3427)

**Comment by Crissman** [comment]
LGTM. Jenkins, test this please.

**Comment by Crissman** [comment]
Merging.

**Pull request merged**